app/src/main/java/com/connorlay/knitgrid/fragments/PatternGridLegendFragment.java

```diff
@@ -0,0 +1,78 @@
+package com.connorlay.knitgrid.fragments;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.Parcelable;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.connorlay.knitgrid.R;
+import com.connorlay.knitgrid.activities.PatternGridViewActivity;
+import com.connorlay.knitgrid.activities.StitchDetailActivity;
+import com.connorlay.knitgrid.adapters.KnitGridAdapter;
+import com.connorlay.knitgrid.models.Pattern;
+import com.connorlay.knitgrid.models.Stitch;
+import com.connorlay.knitgrid.models.StitchPatternRelation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import butterknife.ButterKnife;
+
+/**
+ * Created by Greg on 3/15/2016.
+ */
+public class PatternGridLegendFragment extends Fragment implements KnitGridAdapter.OnStitchRowClickListener{
```

Comment:
I would rename this to something like `PatternDetailLegendFragment` to be consistent with `PatternDetailActivity`.